packages/button/README.md

```diff
@@ -300,11 +337,10 @@ While disabled, `<sp-button>` elements will not respond to click events and will
 
 ### Pending
 
-While in pending state, `<sp-button>` elements will not respond to click events and will appear faded with an indeterminent `<sp-progress-circle>`.
+While in pending state, `<sp-button>` elements will not respond to click events and will appear faded with an indeterminate `<sp-progress-circle>`.
```

Comment:
The pending example should include pendingLabel/pending-label as any usage without it will fail translation